internal/pkg/filesystem/filesystem.go

```diff
@@ -341,11 +393,55 @@ func (f *Fs) ReadJsonMapTo(path, desc string, target interface{}, tag string) (*
 func (f *Fs) ReadFileContentTo(path, desc string, target interface{}, tag string) (*model.File, error) {
 	if field := utils.GetOneFieldWithTag(tag, target); field != nil {
 		if file, err := f.ReadFile(path, desc); err == nil {
-			utils.SetField(field, file.Content, target)
+			content := strings.TrimRight(file.Content, " \r\n\t")
+			utils.SetField(field, content, target)
 			return file, nil
 		} else {
 			return nil, err
 		}
 	}
 	return nil, nil
 }
+
+func newFileError(msg string, file *model.File, err error) error {
+	fileDesc := strings.TrimSpace(file.Desc + " file")
+	if err == nil {
+		return fmt.Errorf("%s %s \"%s\"", msg, fileDesc, file.Path)
+	} else {
+		return fmt.Errorf("%s %s \"%s\": %w", msg, fileDesc, file.Path, err)
+	}
+}
+
+// Rel returns relative path.
+func Rel(base, path string) string {
```

Comment:
Nove funkcie, aby me mali kontrolu nad generovani `path`. Je to iba wrapper nad `filepath`.